rollup.config.js

```diff
@@ -25,14 +25,14 @@ export default {
       jail: `${process.cwd()}/../../`,
       // dedupe: true,
     }),
-    // Minify HTML template literals
-    minifyHTML(),
-    // Minify JS
-    terser({
-      ecma: 2020,
-      module: true,
-      warnings: true,
-    }),
+    // // Minify HTML template literals
+    // minifyHTML(),
+    // // Minify JS
+    // terser({
```

Comment:
Ouch, hopefully we can revert soon

Reply:
part of debugging. will remove in rebase cleanup